src/components/Forms/EditEventForm/EditEventFormComponent.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react'
+import Button from "@material-ui/core/Button";
+import TextField from "@material-ui/core/TextField";
+import Paper from "@material-ui/core/Paper";
+import Typography from "@material-ui/core/Typography"
+import DateFnsUtils from '@date-io/date-fns';
+import {
+    MuiPickersUtilsProvider,
+    KeyboardDateTimePicker
+} from '@material-ui/pickers';
+import { connect } from "react-redux";
+const slugify = require('slugify')
+
+function EditEventFormComponent(props) {
+    const {
+        values: { ename, slug, description, capacity, location, imageUrl, startDate },
+        errors,
+        touched,
+        handleSubmit,
+        handleChange,
+        setFieldValue,
+        setFieldTouched
+    } = props;
+
+    const change = (name, e) => {
+        e.persist();
+        handleChange(e);
+        setFieldTouched(name, true, false);
+    };
+
+    const handleEventNameChange = (name, e) => {
+        e.persist();
+        console.log(props.event)
+        const newSlug = slugify(e.target.value, { lower: true });
+        setFieldValue('slug', newSlug)
+        handleChange(e);
+        setFieldTouched(name, true, false);
+    }
+
+    const handleStartDateChange = (date) => {
+        setFieldValue("startDate", date)
+    }
+
+    const handleEndDateChange = (date) => {
+        setFieldValue("endDate", date)
+    }
+
+    return (
+        <div>
+            <Typography variant="h3">New Event Form</Typography>
+            <form onSubmit={handleSubmit}>
+                <Paper>
+                    <TextField
+                        id="ename"
+                        label={props.event.ename}
```

Comment:
The labels should actually be the same as the new event form
We want the initial values to have the event's previous info, not the labels